concepts/05 Widgets/Form/10 Organize Simple Items/10 In Tabs/05 Columns within a Tab.md

```diff
@@ -79,4 +79,104 @@ The content of a tab can be organized in columns. The [colCount](/api-reference/
         // ...
     })
 
----
\ No newline at end of file
+##### Vue
+
+    <!-- tab: App.vue -->
+    <template>
+        <DxForm
+            :form-data="employee"
+            :col-count="2">
+            <DxSimpleItem data-field="name" />
+            <DxTabbedItem :col-span="2">
+                <DxTab
+                    title="Info"
+                    :col-span="2"
+                    :col-count="3">
+                    <DxSimpleItem data-field="position" />
```

Comment:
There's no `position` in the data object.

Reply:
Fixed